changelog.md

```diff
@@ -1,60 +1,59 @@
 # Changelog
 
-## v0.13.0 t.b.d.
+## v0.13.0 | 2022-09-06
 
 #### New features
 
 * Allow reading multiple files with `lk.CorrelatedStack` (e.g. `lk.CorrelatedStack("image1.tiff", "image2.tiff")`).
-* Added function `Kymo.line_timestamp_ranges()` to obtain the start and stop timestamp of each scan line in a `Kymo`. Please refer to [Confocal images](https://lumicks-pylake.readthedocs.io/en/latest/tutorial/kymographs.html) for more information.
+* Added `CorrelatedStack.export_video()` to export videos to export multi-frame videos to video formats or GIFs.
+* Added support for steps when slicing frames from `CorrelatedStack`s.
+* Added function `Kymo.line_timestamp_ranges()` to obtain the start and stop timestamp of each scan line in a `Kymo`. Please refer to [Confocal images](https://lumicks-pylake.readthedocs.io/en/stable/tutorial/kymographs.html) for more information.
 * Added `Kymo.flip()` to flip a Kymograph along its positional axis.
-* Propagate `Slice` axis labels when performing arithmetic (when possible).
-* It is now possible to pickle `FdFit` objects. Prior to this change, unpickling an `FdFit` would fail since model identification relied on a stored `id` for each of the models used. The `id` of a variable changes whenever a new variable is created however. After this change, each model is associated with a universally unique identifier (uuid) that is used for identification instead. This uuid is serialized with the `Model` and used by `FdFit` thereby preserving their relationship when pickled/unpickled.
-* Added `KymoLineGroup.estimate_diffusion()` to estimate diffusion constants for a group of kymograph traces.
+* Added `KymoTrackGroup.estimate_diffusion()` to estimate diffusion constants for a group of kymograph traces.
 * Include unit in `DiffusionEstimate` dataclass.
-* Allow downloading files directly from Zenodo using `lk.download_from_doi()`. See the [example on Cas9 binding](https://lumicks-pylake.readthedocs.io/en/latest/examples/cas9_kymotracking/cas9_kymotracking.html) for an example of its use.
-* Made piezo tracking functionality public and added [piezo tracking tutorial](https://lumicks-pylake.readthedocs.io/en/latest/tutorial/piezotracking.html).
-* Added support for steps when slicing frames from `CorrelatedStack`s.
-* Added `CorrelatedStack.export_video()` to export videos to export multi-frame videos to video formats or GIFs.
-* Lazily load `data` and `timestamps` for `TimeSeries` data
+* Added `shape` property to `Scan` and `Kymo`.
 * Added possibility to access property `sample_rate` for `TimeSeries` data with constant sample rate.
```

Comment:
I would bump the `sample_rate` for `TimeSeries` all the way to the top since a lot of users requested this.